splitio/javascript-client
# Feature/mget treatments

# JS SDK

## What did you accomplish?
= Added `mget` for multiple evaluations at once.
- Added new method `evaluateFeatures` for doing multiple evaluations at once.
- Refactored `evaluator` for evaluating one feature and multiple features.

## How do we test the changes introduced in this PR?
- Check redis calls - it should execute one call instead of one per each feature.

## Extra Notes
